tools/rotate-certs.sh

```diff
@@ -69,3 +69,12 @@ ${_kubectl} delete pods --namespace "${namespace}" -l kubevirt.io
 echo "# Rotating SSP certificates ..."
 ${_kubectl} delete secrets --ignore-not-found=true --namespace "${namespace}" virt-template-validator-certs
 ${_kubectl} delete pods --namespace "${namespace}" -l kubevirt.io=virt-template-validator
+
+echo "# Rotating Web UI certificates ..."
+formerVersion=$(${_kubectl} get kwebui kubevirt-web-ui -o yaml | grep '  version: '|sed 's/^.*: *\(.*\)$/\1/g')
+echo Detected former Web UI version: ${formerVersion}
+${_kubectl} patch kwebui kubevirt-web-ui --patch '{"spec": {"version": ""}}' --type=merge # undeploy
+sleep 15 # give kubevirt-web-ui-operator enough time to react, but it's usually instant
```

Comment:
The web-ui-operator needs to start reconciliation (means undeployment). We do not need to wait till it's finished.